java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java

```diff
@@ -37,6 +37,50 @@
     private RpcClient rpcClient;
     private TaskManagerInterface taskManagerInterface;
 
+    public TaskManagerClient(String zkCluster, String zkPath) {
+        Log LOG = LogFactory.getLog(TaskManagerClient.class);
+        if (zkCluster == null || zkCluster.length() == 0 || zkPath == null || zkPath.length() == 0) {
+            LOG.info("Zookeeper address is wrong, please check the configuration");
+        }
+        String masterZnode = zkPath + "/taskmanager" + "/leader";
+        String msg;
+        HostPort hostPort = new HostPort(TaskManagerConfig.HOST, TaskManagerConfig.PORT);
+        CuratorFramework zkClient = CuratorFrameworkFactory.builder() //
+                .connectString(zkCluster)   //连接地址
+                .sessionTimeoutMs(10000)    //超时时间
+                .retryPolicy(new ExponentialBackoffRetry(1000, 10))   //重试策略
+                .build();//
+        zkClient.start();
+
+        try {
```

Comment:
constructor should not contain heavy works, how about move to separate function `init()`

Reply:
I will put part of work into a separate init () function.

Reply:
I mean, do not do call `init` inside constructor.